.github/workflows/linux.yml

```diff
@@ -260,16 +260,17 @@ jobs:
           cmake -DCMAKE_BUILD_TYPE=Release -S ./ -B ./build/
           cmake --build ./build/ --config Release -j
 
-      - name: test bindings
+      - name: Test bindings
         run: |
           source ${OV_INSTALL_DIR}/setupvars.sh
           python -m pip install ./thirdparty/openvino_tokenizers/[transformers] -r ./tests/python_tests/requirements.txt --find-links ${OV_INSTALL_DIR}/tools --upgrade-strategy eager
           python -m pytest ./tests/python_tests
         env:
           PYTHONPATH: "./build/:$PYTHONPATH"
 
-      - name: test bindings (wheel)
+      - name: Test bindings (wheel)
         run: |
+          source ${OV_INSTALL_DIR}/setupvars.sh
```

Comment:
please, drop this line

Reply:
we can't because we need to set OpenVINO env during the wheel build

Reply:
it builds and install it:
Building wheels for collected packages: openvino-genai
  Building wheel for openvino-genai (pyproject.toml): started
  Running command Building wheel for openvino-genai (pyproject.toml)
  Building wheel for openvino-genai (pyproject.toml): finished with status 'done'
  Created wheel for openvino-genai: filename=openvino_genai-2024.4.0.0-cp38-cp38-linux_x86_64.whl size=1[508](https://github.com/openvinotoolkit/openvino.genai/actions/runs/10196424457/job/28207653789#step:11:509)536 sha256=87e1f4b3bc1b54fd584275686aaebf6fffd7d7cc706db9cbe4f1851b06f0a1ca
  Stored in directory: /tmp/pip-ephem-wheel-cache-7qxk_eay/wheels/1b/b2/ae/cdcc204ed399c5f28832598137a73f7de722d4645acd4c8969
Successfully built openvino-genai
Installing collected packages: openvino-genai
Successfully installed openvino-genai-2024.4.0.0

Reply:
That means the the tests must be run in a separate step

Reply:
you are free to split steps as you want, lets do it separately